### ISSUE TITLE
additional data structure for HSN

### ISSUE THREAD

**Issue by rijpma** [CLARIAH/wp4-requirements#125]
Expand HSN data structure to make the structure more accessible and easier to query. Follow-up on https://github.com/CLARIAH/wp4-requirements/issues/63

**Comment by rlzijdeman** [comment]
Can I read this as: add IDS as data strucure?

On 22 May 2017, at 19:36, rijpma <notifications@github.com<mailto:notifications@github.com>> wrote:


Expand HSN data structure to make the structure more accessible and easier to query. Follow-up on #63<https://github.com/CLARIAH/wp4-requirements/issues/63>

-

**Comment by rijpma** [comment]
No, because HSN-IDS is not yet available to us. Some of the additional structure might recreate IDS-functionality though.

You can see queries using the additions here:
https://github.com/CLARIAH/wp4-queries/commit/3d71916e6148905da68da9243a3ea9d960640e67
https://github.com/CLARIAH/wp4-queries/commit/0011aa6053f003f75624087d5ac92ba0bf5744d3
https://github.com/CLARIAH/wp4-queries/commit/511af66a4aa77d3cba8b89dad03f5ceeb5caf91a

I've assigned it to the three of us to get some feedback on my design choices here. Any comments are welcome, be it small things like style or big things like graph structure.
